### PR TITLE
Default registry

### PR DESCRIPTION
**Pull request by skygeo**
Add DEFAULT_DOCKER_REGISTRY environment variable

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,3 +6,6 @@ The following environment variables are required to run the plugin:
 * FLYBY_ENDPOINT
 * GROUNDCONTROL_ENDPOINT
 * BOSUN_ENDPOINT
+
+Aslo, the following environment might be added:
```
Comment (balintzs) [inline README.md]: "Also"

File: main.py [hunk 2]
```diff
@@ -168,6 +169,12 @@ def rollback(payload):
 def deploy(payload):
 
     steps = []
+    # Set default docker registry
+    default_docker_registry = os.getenv("DEFAULT_DOCKER_REGISTRY")
+    if default_docker_registry:
+        for container in payload['task_definition']['container_definitions']:
+            if len(re.findall('/', container['image'])) < 2:
+                container['image'] = "{}/{}".format(default_docker_registry, container['image'])
```
Comment (balintzs) [inline main.py]: What if the image doesn't have a namespace, like `hub.docker.com/foo`?

Comment (skygeo) [inline main.py]: Looking at this, seems that @gnhenderson  is right... dot is what to look for. 
https://github.com/docker/distribution/blob/master/reference/regexp.go#L25

 

@balintzs foo/bar ... how do we know if foo is

**Comment by balintzs** [comment]
LGTM

**Pull request merged**